pulser-pasqal/pulser_pasqal/backends.py

```diff
@@ -223,20 +223,55 @@ def __init__(
             config=config,
             mimic_qpu=mimic_qpu,
         )
-        # To be deleted once this PR is released: https://github.com/pasqal-io/Pulser/pull/890
-        self._config = type(self.default_config)(
-            **{
-                **self.default_config._backend_options,
-                **(config._backend_options if config else {}),
-            }
-        )
+
+    def run(
+        self, job_params: list[JobParams] | None = None, wait: bool = False
+    ) -> RemoteResults:
+        """Runs the sequence on this remote emulator and returns the result.
+
+        Args:
+            job_params: A list of parameters for each job to execute. If the
+                sequence is parametrized, the values for all the variables
+                necessary to build the sequence must be given for
+                each job, under the 'variables' field. If not given, a single
+                job is executed.
+            wait: Whether to wait until the results of the jobs become
+                available.  If set to False, the call is non-blocking and the
+                obtained results' status can be checked using their `status`
+                property.
+
+        Warning:
+            Unlike a 'QPUBackend', this backend does not expect a value for
+            "runs" in each entry of 'job_params'. If provided, this value is
+            ignored. If you wish to set the total number of bitstring counts
+            in the Results, please provide a 'BitStrings' observable with the
+            desired 'num_shots' via this backend's 'config' instead."
+
+        Returns:
+            The results, which can be accessed once all sequences have been
+            successfully executed.
+        """
+        _job_params: list[JobParams]
+        if job_params is None:
+            # Assume a single job
+            _job_params = [{"runs": 1}]
```

Comment:
We could make this optional on the cloud side so that this doesn't need to be explicitly set with a placeholder value. What do you think?

Reply:
My suggestion was to not set any JobParams at all in this`RemoteEmulatorBackend` and pass it as is in `PasqalCloud.submit()`. This requires a small change in our APIs to accept empty job params for emulators but it shouldn't be difficult. The main benefit for that is that we don't send placeholders values which are ignored throughout the stack and may be confusing for users/debugging

Reply:
Just so we are on the same page for future developments, what's currently missing to enable job_params to be None (instead of the placeholder `{"runs": 1}`)  for emulators? 

1) Cloud API validation update to allow None values for non-legacy emulators
2) changes to the type hints in pulser_pasqal for the run function.

Are there changes to do on pulser side as well?